### PR TITLE
EWPP-1914: Update to PHP8

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,31 +6,41 @@
...
+        "openeuropa/open_vocabularies": "^1.0-alpha4",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0.0-beta5",
+        "phpspec/prophecy-phpunit": "^1 || ^2",
+        "symfony/dom-crawler": "^4.4.12"
     },
+    "_readme": [
+        "Explicit lower version requirement of ctools due to Drupal core 9.2 compatibility.",
+        "Explicit requirement for egulias/email-validator due to https://www.drupal.org/project/drupal/issues/3061074#comment-14300579. It can be removed when Drupal core 9.2 support is droppped.",
+        "Explicit minimum version requirement for symfony/dom-crawler due to its lower versions using the deprecated function libxml_disable_entity_loader() in PHP8.",
+        "Explicit requirement of drupal/token module due to PHP 8 compatbility."
```
Comment (22Alexandra) [inline composer.json:42]: Missing readme entry for some of the new dependencies such as ctools, symfony/dom-crawler.

File: composer.json [hunk 2]
```diff
@@ -6,31 +6,41 @@
...
+        "drush/drush": "^10.3",
+        "egulias/email-validator": "^2.1.22 || ^3.0",
+        "openeuropa/code-review": "^2.0",
         "openeuropa/oe_link_lists": "^0.11",
-        "openeuropa/oe_multilingual": "^1.8",
-        "openeuropa/open_vocabularies": "^1.0-alpha4"
+        "openeuropa/oe_multilingual": "^1.10",
+        "openeuropa/open_vocabularies": "^1.0-alpha4",
+        "openeuropa/task-runner-drupal-project-symlink": "^1.0.0-beta5",
+        "phpspec/prophecy-phpunit": "^1 || ^2",
```
Comment (22Alexandra) [inline composer.json:35]: We can use only ^2, no?

File: composer.json [hunk 3]
```diff
@@ -6,31 +6,41 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=7.3",
-        "drupal/core": "^8.9 || ^9.1",
+        "php": ">=7.4",
+        "drupal/core": "^9.2",
         "drupal/emr": "^1.0-beta11",
```
Comment (22Alexandra) [inline composer.json:11]: Use the dev branch where the upgrade has been done in order to test it.
The same for the multivalue_form_element component.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
Update core version requirement in oe_list_pages_open_vocabularies_test.info.yml

File: composer.json (hunk 1 above)
Comment (yenyasinn) [reply composer.json:42]: All new dependencies have been added to readme including ctools and symfony/dom-crawler. What have I missed?

File: composer.json (hunk 2 above)
Comment (yenyasinn) [reply composer.json:35]: We can use only ^2 or leave "phpspec/prophecy-phpunit": "^1 || ^2" as in other components.

File: composer.json (hunk 1 above)
Comment (22Alexandra) [reply composer.json:42]: You're right. My bad.

File: composer.json (hunk 2 above)
Comment (22Alexandra) [reply composer.json:35]: True, let's use only ^2 :)

**Review by 22Alexandra — DISMISSED** [review]

**Review by nagyad — APPROVED** [review]

**Review by hernani — APPROVED** [review]

**Review by nagyad — APPROVED** [review]

**Pull request merged**